process_request/views.py

```diff
@@ -47,6 +47,32 @@ def post(self, request):
             return Response({"detail": str(e)}, status=500)
 
 
+class DeliverReadingRoomRequestView(APIView):
+    "Delivers a request for records to be delivered to the reading room."
```

Comment:
Should this and line 64 be comments?

Reply:
Oh, good catch. This should actually probably be docstrings, right @helrond?

Reply:
Good catch - yup, those are docstrings. Updated.